mkdocs.yml

```diff
@@ -21,14 +21,15 @@ nav:
   - Configuration Options: config.md
   - Authoring Rules: rules-authoring.md
   - Available Rules: rules-index.md
-  - FAQ: faq.md
   - Architecture: architecture.md
   - Tracing:
     - Getting Started: tracee-ebpf/index.md
     - Output Options: tracee-ebpf/output.md
     - Trace Options: tracee-ebpf/trace-options.md
     - Capturing Artifacts: tracee-ebpf/capture.md
     - Overriding OS needed files: tracee-ebpf/override-os-needed-files
+  - FAQ: faq.md
+  - Further Reading: further-reading.md
```

Comment:
Could we put "Further Reading" within a section called "References"? We are trying to follow this framework and this would be a great first step to implementing it 💯 https://diataxis.fr/

Reply:
I really like the framework but I think reference documentation related to something else, also in the framework explanation. 

I think they don't mean "references" like extra things you could check out. Rather "the reference documentation" like the complete description of all the functionalities and options.

In the doc they say that reference documentation is "dry information" like "encyclopedia article"

Reply:
Yes, you are right.
What do you think of having a community section in each docs?
E.g. the structure could look something like this

![docs](https://user-images.githubusercontent.com/33576047/154669679-166ff78c-609a-4acb-869f-b839eaa84707.png)

Reply:
personally I  think that the best option is to add these external link in the tutorials or how-to sections. If someone wrote a good tutorial, we should make it discoverable in the docs right where all other tutorials are.

Reply:
I think that most tutorials are written in blogs, not in the docs, so the docs' tutorials section will be almost exclusively external links anyway (also we do this, see the aqua blog)

Reply:
@itaysk tutorials in the tutorial section in the docs have a different purpose to tutorials in a blog -- usually you provide additional explanations in blog posts that should not be part of a tutorial in the docs 
Separately, I like the suggestion of highlighting external tutorials in the tutorials section